[Bug Skip]: test_ops_xpu.py TestCommonXPU compare_cpu frexp AssertionError tensor-likes not close

### 🐛 Describe the bug

Multiple `test_ops_xpu.py` `compare_cpu_frexp` cases fail on Windows with Intel XPU (Arc B580) with `AssertionError: Tensor-likes are not close!`.

### Affected Test Cases

```
test_ops_xpu.py::TestCommonXPU::test_compare_cpu_frexp_xpu_bfloat16
test_ops_xpu.py::TestCommonXPU::test_compare_cpu_frexp_xpu_float16
test_ops_xpu.py::TestCommonXPU::test_compare_cpu_frexp_xpu_float32
test_ops_xpu.py::TestCommonXPU::test_compare_cpu_frexp_xpu_float64
```

### Error Message

<details>
<summary>Click to expand traceback</summary>

```
Traceback (most recent call last):
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\unittest\case.py", line 58, in testPartExecutor
    yield
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\unittest\case.py", line 634, in run
    self._callTestMethod(testMethod)
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\unittest\case.py", line 589, in _callTestMethod
    if method() is not None:
       ^^^^^^^^
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\site-packages\torch\testing\_internal\common_utils.py", line 3535, in wrapper
    method(*args, **kwargs)
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\site-packages\torch\testing\_internal\common_device_type.py", line 535, in instantiated_test
    result = test(self, **param_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\site-packages\torch\testing\_internal\common_utils.py", line 1928, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\site-packages\torch\testing\_internal\common_device_type.py", line 1296, in test_wrapper
    raise e
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\site-packages\torch\testing\_internal\common_device_type.py", line 1283, in test_wrapper
    return test(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\site-packages\torch\testing\_internal\common_device_type.py", line 1580, in only_fn
    return fn(slf, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\site-packages\torch\testing\_internal\common_utils.py", line 2658, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\site-packages\torch\testing\_internal\common_utils.py", line 2630, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\gta\repositories\pytorch\pytorch\third_party\torch-xpu-ops\test\xpu\extended\test_ops_xpu.py", line 167, in test_compare_cpu
    test_common_test_fn(self.proxy, device, dtype, op)
  File "C:\Users\gta\repositories\pytorch\pytorch\third_party\torch-xpu-ops\test\xpu\extended\test_ops_xpu.py", line 143, in test_compare_cpu
    self.assertEqual(xpu_results, cpu_results, atol=1e-3, rtol=1e-3)
  File "C:\Users\gta\miniforge3\envs\202605151131_9ed8fd28_32.0.101.8801_2025.3.2.61\Lib\site-packages\torch\testing\_internal\common_utils.py", line 4553, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Tensor-likes are not close!

Mismatched elements: 9 / 9 (100.0%)
Greatest absolute difference: 1 at index (0,) (up to 0.001 allowed)
Greatest relative difference: 1.0 at index (0,) (up to 0.001 allowed)

```

</details>


### Versions

<details>
<summary>Click to expand environment details</summary>

```
PyTorch version: 2.13.0a0+git9ed8fd2
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: Microsoft Windows 11 Pro (10.0.26100 64-bit)
GCC version: Could not collect
Clang version: Could not collect
CMake version: version 3.31.6
Libc version: N/A

Python version: 3.12.13 | packaged by conda-forge | (main, Mar  5 2026, 16:36:12) [MSC v.1944 64 bit (AMD64)] (64-bit runtime)
Python platform: Windows-11-10.0.26100-SP0
Is CUDA available: False
CUDA runtime version: No CUDA
CUDA_MODULE_LOADING set to: N/A
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
Is XPU available: True
XPU used to build PyTorch: 20250302
Intel GPU driver version:
* 32.0.101.8801 (20260512000000.******+***)
Intel GPU models onboard:
* Intel(R) Arc(TM) B580 Graphics
Intel GPU models detected:
* [0] _XpuDeviceProperties(name='Intel(R) Arc(TM) B580 Graphics', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero V2', type='gpu', device_id=0xE20B, uuid=86800be2-0000-0000-0400-000000000000, driver_version='1.15.37858', total_memory=11875MB, local_mem_size=128KB, max_compute_units=160, memory_clock_rate=0MHz, memory_bus_width=64-bit, gpu_eu_count=160, gpu_subslice_count=20, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: True
Caching allocator config: N/A

CPU:
Name: Intel(R) Core(TM) Ultra 5 245K
Manufacturer: GenuineIntel
Family: 773
Architecture: 9
ProcessorType: 3
DeviceID: CPU0
CurrentClockSpeed: 3071
MaxClockSpeed: 4200
L2CacheSize: 26624
L2CacheSpeed: None
Revision: None

Versions of relevant libraries:
[pip3] bert_pytorch==0.0.1a4
[pip3] functorch==1.14.0a0+b71aa0b
[pip3] intel-openmp==2025.3.3
[pip3] mkl-include==2025.3.1
[pip3] mkl-static==2025.3.1
[pip3] mypy==2.1.0
[pip3] mypy_extensions==1.1.0
[pip3] numpy==1.26.2
[pip3] onemkl-license==2025.3.1
[pip3] onnx==1.21.0
[pip3] onnx-ir==0.1.16
[pip3] onnxscript==0.6.2
[pip3] optree==0.13.0
[pip3] pytorch-labs-segment-anything-fast==0.2
[pip3] tbb==2022.3.1
[pip3] tbb-devel==2022.3.1
[pip3] tcmlib==1.4.1
[pip3] torch==2.13.0a0+git9ed8fd2
[pip3] torch_geometric==2.4.0
[pip3] torchao==0.17.0
[pip3] torchaudio==2.11.0a0+c0cbdb9
[pip3] torchbench==0.1
[pip3] torchmetrics==1.9.0
[pip3] torchmultimodal==0.1.0b0
[pip3] torchrec-nightly==2022.4.26
[pip3] torchvision==0.27.0a0+499ca51
[pip3] torchx-nightly==2026.5.15
[pip3] triton-xpu==3.7.1+git21033c4e
[conda] bert-pytorch              0.0.1a4                   dev_0    <develop>
[conda] functorch                 1.14.0a0+b71aa0b          pypi_0    pypi
[conda] intel-openmp              2025.3.3                 pypi_0    pypi
[conda] mkl-include               2025.3.1                 pypi_0    pypi
[conda] mkl-static                2025.3.1                 pypi_0    pypi
[conda] numpy                     1.26.2                   pypi_0    pypi
[conda] onemkl-license            2025.3.1                 pypi_0    pypi
[conda] optree                    0.13.0                   pypi_0    pypi
[conda] pytorch-labs-segment-anything-fast 0.2                      pypi_0    pypi
[conda] tbb                       2022.3.1                 pypi_0    pypi
[conda] tbb-devel                 2022.3.1                 pypi_0    pypi
[conda] tcmlib                    1.4.1                    pypi_0    pypi
[conda] torch                     2.13.0a0+git9ed8fd2          pypi_0    pypi
[conda] torch-geometric           2.4.0                    pypi_0    pypi
[conda] torchao                   0.17.0                   pypi_0    pypi
[conda] torchaudio                2.11.0a0+c0cbdb9          pypi_0    pypi
[conda] torchbench                0.1                      pypi_0    pypi
[conda] torchmetrics              1.9.0                    pypi_0    pypi
[conda] torchmultimodal           0.1.0b0                  pypi_0    pypi
[conda] torchrec-nightly          2022.4.26                pypi_0    pypi
[conda] torchvision               0.27.0a0+499ca51          pypi_0    pypi
[conda] torchx-nightly            2026.5.15                pypi_0    pypi
[conda] triton-xpu                3.7.1+git21033c4e          pypi_0    pypi
```

</details>

--- a/test/xpu/extended/skip_list_win.py
+++ b/test/xpu/extended/skip_list_win.py
@@ -11,9 +11,6 @@ skip_dict = {
         "test_compare_cpu_pow_xpu_bfloat16",  # https://github.com/intel/torch-xpu-ops/pull/764
         "test_compare_cpu_argmin_xpu_int",
         # https://github.com/intel/torch-xpu-ops/issues/3805
-        "test_compare_cpu_frexp_xpu_bfloat16",
-        "test_compare_cpu_frexp_xpu_float16",
-        "test_compare_cpu_frexp_xpu_float32",
-        "test_compare_cpu_frexp_xpu_float64",
+        "test_compare_cpu_frexp_xpu",
     ),
 }

--- a/test/xpu/extended/skip_list_win.py
+++ b/test/xpu/extended/skip_list_win.py
@@ -10,5 +10,10 @@ skip_dict = {
     "test_ops_xpu.py": (
         "test_compare_cpu_pow_xpu_bfloat16",  # https://github.com/intel/torch-xpu-ops/pull/764
         "test_compare_cpu_argmin_xpu_int",
+        # https://github.com/intel/torch-xpu-ops/issues/3805
+        "test_compare_cpu_frexp_xpu_bfloat16",
+        "test_compare_cpu_frexp_xpu_float16",
+        "test_compare_cpu_frexp_xpu_float32",
+        "test_compare_cpu_frexp_xpu_float64",
     ),
 }

